src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/OutlineAccessibility.m

```diff
@@ -55,4 +55,88 @@ - (NSString *)accessibilityLabel
     return [[super accessibilityLabel] isEqualToString:@"list"] ? @"tree" : [super accessibilityLabel];
 }
 
+- (nullable NSArray<id<NSAccessibilityRow>> *)accessibilityRows
+{
+    return [self accessibilityChildren];
+}
+
+- (nullable NSArray<id<NSAccessibilityRow>> *)accessibilitySelectedRows
+{
+    return [self accessibilitySelectedChildren];
+}
+
+- (nullable  NSArray<id<NSAccessibilityRow>> *)accessibilityChildren
+{
+    if (![self isCacheValid]) {
```

Comment:
Why can't we use `rowCacheValid` value to check like it is done for `accessibilitySelectedChildren` method?

Reply:
Again - to cover the situation where expand/collapse events were delivered to the parent node. Or to the parent's parent.